src/autopas/containers/linkedCells/LinkedCellsReferences.h

```diff
@@ -313,7 +312,8 @@ class LinkedCellsReferences : public CellBasedParticleContainer<ReferenceParticl
 
         auto &particleVec = this->getCells()[cellId]._particles;
         for (auto pIter = particleVec.begin(); pIter != particleVec.end();) {
-          if (utils::notInBox((*pIter)->getR(), cellLowerCorner, cellUpperCorner)) {
+          if ((*pIter)->getOwnershipState() == OwnershipState::owned and
+              utils::notInBox((*pIter)->getR(), cellLowerCorner, cellUpperCorner)) {
```

Comment:
```suggestion
          if ((*pIter)->isOwned() and utils::notInBox((*pIter)->getR(), cellLowerCorner, cellUpperCorner)) {
```
Keep it simple ;)

Reply:
done in [06acfa6](https://github.com/AutoPas/AutoPas/pull/794/commits/06acfa6ec157f4d573d6d3b989f242caf776aef8)